platform-includes/user-feedback/sdk-api-example/flutter.mdx

```diff
@@ -0,0 +1,26 @@
+```dart
+// Option 1: Retrieving SentryId from beforeSend
+SentryId sentryId = SentryId.empty();
+
+await SentryFlutter.init((options) {
+  options.beforeSend = (event, hint) async {
+    sentryId = event.eventId;
+    return event;
+  };
+});
+
+// Option 2: Retrieving SentryId from the method capturing the event
+SentryId sentryId = Sentry.captureMessage("My message");
+
+// Option 3: Retrieving SentryId from the beforeSend callback
+SentryId sentryId = Sentry.lastEventId;
+
+final userFeedback = SentryUserFeedback(
+    eventId: sentryId,
+    comments: 'Hello World!',
+    email: 'foo@bar.org',
+    name: 'John Doe',
+);
+
+Sentry.captureUserFeedback(userFeedback);
+```
```

Comment:
will we need this? I think we are going to update the feedback docs anyway: https://github.com/getsentry/sentry-docs/issues/11756

Reply:
just added it to be in line with the others. we can remove/update then once the other docs land

Reply:
okay then let's change this when we're doing the new user feedback